pkg/kapp/cmd/app/delete.go

```diff
@@ -95,13 +100,21 @@ func (o *DeleteOptions) Run() error {
 		return err
 	}
 
+	// Deciding based on isFullyDeleteFlag flag from resource filter and diff filter
```

Comment:
before this if, i would recommend to reset isFullyDeleteApp to something like

```
if changesSummary.SkippedChanges {
  isFullyDeleteApp = false
}
```

also i would recommend renaming it back to `fullyDeleteApp`, or to `shouldFullyDeleteApp`

Reply:
I believe instead of reseting `isFullyDeleteApp` flag now we can change the condition at 104 to something like this - 
```
if !shouldFullyDeleteApp || changesSummary.SkippedChanges {
		shouldFullyDeleteApp = false
		o.ui.PrintLinef("App '%s' (namespace: %s) will not be fully deleted "+
			"because some resources are excluded by filters",
			app.Name(), o.AppFlags.NamespaceFlags.Name)
}
```
Here I renamed it to `shouldFullyDeleteApp`